.github/CONTRIBUTING.md

```diff
@@ -26,15 +26,32 @@ Generally speaking, you should fork this repository, make changes in your own fo
 ### Working with forks
 
 * [Configure this repository as a remote for your own fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and
+
+```shell
```

Comment:
Why should we explicitly describe a standard github working mode here? Developers already working with github for some time are supposed to know this anyway.

For the others there are already tutorials, e.g. [here](https://help.github.com/en/github/getting-started-with-github/fork-a-repo). Developers not knowing this working mode should be able to find tutorials like that using a search engine of their choice.

IMO we are repeating the obivous here.